### PR TITLE
Use 'build-base' so the core18 snap is not required when installing

### PR DESCRIPTION
**Pull request by townsend2010**

### CODE REVIEW

File: snapcraft.yaml [hunk 1]
```diff
@@ -8,7 +8,8 @@ license: GPL-2.0
 
 version: '0.1'
 confinement: strict
-base: core18
+type: base
```
Comment (Saviq) [inline snapcraft.yaml:11]: We should instead have this:

```suggestion
base: bare
```

**Review by Saviq — CHANGES_REQUESTED** [review]
We can't be a `type: base` snap, as that would mean someone would be able to do `base: multipass-sshfs` in their snap, which doesn't make sense.

Snapcraft might not support `base: bare; build-base: core18` just yet, but we should be able to manually amend the snap metadata before uploading to the store while we wait for this to be supported.

**Comment by townsend2010** [comment]
So, I tried directly modifying the `snap.yaml` file in the unsquashed snap.

With `base: bare` and then try to install the snap:
```
error: cannot perform the following tasks:
- Ensure prerequisites for "multipass-sshfs" are available (no snap revision available as specified)
```

With `base: none` and then try to install the snap:
```
error: cannot perform the following tasks:
- Ensure prerequisites for "multipass-sshfs" are available (snap not found)
```

I think at this point, the only way to build a snap without requiring the base is to make it a base snap itself, and as has been said, that's not really what we want.

**Comment by Saviq** [comment]
> ```
> error: cannot perform the following tasks:
> - Ensure prerequisites for "multipass-sshfs" are available (no snap revision available as specified)
> ```

This is fine! It's just that the `bare` snap is only available on `edge` today.

```
$ snap info bare
…
channels:
  stable:    –                      
  candidate: –                      
  beta:      –                      
  edge:      1.0 2017-07-26 (5) 4kB -
```

We just need to bring the bare snap with us as well. Which was the plan all along if we want to ensure offline operation.

Oh and just realized, if we bring the snap along with its assertion (`snap download <snap>`), it will know straight away that it can upgrade from the store :)

**Comment by townsend2010** [comment]
This is no longer needed so closing.

**Pull request closed without merging**